src/ParkingAPI.java

```diff
@@ -51,4 +51,16 @@ public ParkingLot availableParkingLots(ParkingLot one, ParkingLot two) {
 			return null;
 		}
 	}
+	public ParkingLot predictedOpenSpots(ParkingLot one, ParkingLot two) {
+		int lotOne = getPredictedOpenSpots(one) - getPredictedClosedSpots(one);
+		int lotTwo = getPredictedOpenSpots(two) - getPredictedClosedSpots(two);
```

Comment:
Once again, I think you messed up the logic again in the exact same way as last time. I assume that you just want to get the predicted open spots, which should just be `getPredictedOpenSpots()`.

case example: Prediction states that there will be 2 open spots of the 100 total spots. `getPredictedOpenSpots() - getPredictedClosedSpots()` would return `-98`

Reply:
Yes, that's what I tried to implement, but I realized that you already have that taken care of.. .

Reply:
Should fix it, although I feel as though we may not even need the printlns in availableParkingLots and redictedOpenSpots methods...